libraries.json

```
@@ -574,5 +574,18 @@
     "source": "youritjang/azure-cloud-services.excalidrawlib",
     "preview": "youritjang/azure-cloud-services.png",
     "date": "2021-06-03"
+  },
+  {
+    "name": "Customer Journey Map",
+    "description": "For creating your own Customer Journey Map, it has small and big templates.",
+    "authors": [
+      {
+        "name": "Wiktoria Mielcarek / Braweria",
+        "url": "https://github.com/braweria"
+      }
+    ],
+    "source": "braweria/Customey-Journey-Map.excalidrawlib ",
+    "preview": "libraries/braweria/CustomeyJourneyMap.png",
```

```suggestion
    "preview": "braweria/CustomeyJourneyMap.png",
```

should be without the leading `libraries/`. I'd also maybe fix the `Customey` typo? Otherwise :+1:

---

thank you.